README.md

```diff
@@ -24,11 +24,11 @@ What does it do?
 
 |   | macOS Intel | macOS Apple Silicon | Windows 64bit | Windows 32bit | manylinux x86_64 | manylinux i686 | manylinux aarch64 | manylinux ppc64le | manylinux s390x |
 |---|---|---|---|---|---|---|---|---|---|
-| CPython 3.6     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
-| CPython 3.7     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
-| CPython 3.8     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
+| CPython 3.6     | ✅ | N/A | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
+| CPython 3.7     | ✅ | N/A | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
+| CPython 3.8     | ✅ | N/A | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
```

Comment:
Evidently it's more complicated than "N/A" here: https://twitter.com/hugovk/status/1397275649253990405

Reply:
Yeah, essentially, since we only have Intel CI runners running 10.15, we can't tick this box. But the wheels could exist, if you were able to build on an Apple Silicon machine.

Reply:
> But the wheels could exist, if you were able to build on an Apple Silicon machine.
or x86_64 runners with macOS 11+

Reply:
GitHub Actions was working on that a while back.

Reply:
I'd put a superscript here, and have a little note. Since there is a Python 3.8 Universal release _and_ 3.8 is the default Python currently with Big Sur's XCode toolkit.

Reply:
How about:

```
CPython 3.8's final binary release has experimental Universal2 support,
but does not support macOS 10.x, so this is not currently available in cibuildwheel.
```

If we were running on macOS 11 runners, are you pretty sure we could get this to work?

Reply:
Should we add add N/A For PyPy and the special arches, as those are not provided by PyPy & manylinux?

Reply:
@pypa/cibuildwheel-team opinions here on these two choices? The comment and the N/As for PyPy? (Maybe @mattip too on PyPy)

Reply:
N/A would be appropriate I think.